tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -166,6 +169,19 @@ Verify RHODS Release Version Number
     ${version} =  Get RHODS Version
     Should Match Regexp    ${version}    ^[0-9]+\.[0-9]+\.[0-9]+\(-[0-9]+)*$
 
+Verify That Alert Notification Emails Can Be Updated And Changed In Alertmanager Configuration
+    [Documentation]    Vrifies the Alert Notification email is updated in Addon-Managed-Odh-Parameters Secret and Alertmanager ConfigMap
+    [Tags]    Sanity
+    ...       Tier2
+    ...       ODS-673
+    ...       KnownIssues
+    ${email_to_change} =    Set Variable    dummyemail2@redhat.com
+    ${cluster_id} =    Get Cluster ID
+    ${cluster_name} =    Get Cluster Name By Cluster ID    ${cluster_id}
+    Update Notification Email Address    ${cluster_name}    ${email_to_change}
+    ${email_from_secret} =    Get Notification Email From Addon-Managed-Odh-Parameters Secret
```

Comment:
The known bug in RHODS is not this one.  Notification email should have the new value. You probably need to create a keyword  `Wait Until Notification Email From Addon-Managed-Odh-Parameters Contains $email  $retries`